dataland-backend/src/main/kotlin/org/dataland/datalandbackend/repositories/DataMetaInformationRepository.kt

```diff
@@ -51,4 +51,18 @@ interface DataMetaInformationRepository : JpaRepository<DataMetaInformationEntit
         @Param("dataType") dataType: String,
         @Param("reportingPeriod") reportingPeriod: String,
     ): DataMetaInformationEntity?
+
+    /**
+     * Counts the dataset meta information
+     * filtered by company ID, data type and if it is currently active
+     * @param company_companyId the ID of the company to filter for
+     * @param dataType the data type to filter for
+     * @param currentlyActive the currently active filter
+     * @returns the number of data meta informations that fulfill these criteria
+     */
+    fun countByCompany_CompanyIdAndDataTypeAndCurrentlyActive(
```

Comment:
Wie wäre es mir `countDatasetsByCompanyIdAndDataType`

Reply:
It is no called countByCompanyIdAndDataTypeAndCurrentlyActive. 
Is this ok for you?